splinterd/sample_configs/splinterd.toml.example

```diff
@@ -13,40 +13,95 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
+# The default values are shown for each setting, unless otherwise noted.
+
 # Config file format version
 version = "1"
 
+# Sets a new ID for the node. The node ID must be unique across the
+# network (for all Splinter nodes that could participate on the same circuit).
+# (default 'nXXXXX', where 'XXXXX' is 5 random numbers)
 node_id = "012"
 
+# Human-readable name for the node
+# (default 'Node {node_id}')
+display_name = "node-012"
+
 # Endpoint used for service to daemon communication.
+# (default "127.0.0.1:8043")
 service_endpoint = "127.0.0.1:8043"
 
 # Endpoints used for daemon to daemon communication.
+# (default ["127.0.0.1:8044"])
```

Comment:
for network_endpoint and peers, please add the format of the endpoints. (protocl_prefix://<endpoint>)

Reply:
Addressed